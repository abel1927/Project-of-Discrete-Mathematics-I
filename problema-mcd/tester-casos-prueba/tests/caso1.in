5
7 13
14 23
11 19
11 12
13 21
