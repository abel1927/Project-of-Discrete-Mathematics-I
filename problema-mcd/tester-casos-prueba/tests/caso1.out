12
22
18
4
12
